subgraph.yaml

```diff
@@ -1,114 +1,47 @@
-        - event: PropertiesSet(indexed address,indexed uint256,string)
-          handler: handlePropertiesSet
-        - event: ControllerSet(indexed uint256,indexed address,indexed address,address)
-          handler: handleControllerSet
-        - event: ChildLockablePropertiesRuleSet(indexed address,indexed uint256,uint8)
-          handler: handleChildLockablePropertiesRuleSet
-
-      file: ./src/registry.ts
-  - kind: ethereum/contract
-    name: StakingController
-    network: rinkeby
-    source:
-      address: "0x798190f0640E6089b481298FbAc7EAEC0CFFBEFb"
-      abi: StakingController
-      startBlock: 7982879
-    mapping:
-      kind: ethereum/events
-      apiVersion: 0.0.4
-      language: wasm/assemblyscript
-      entities:
-        - Domain
-        - StakingDomainConfig
-        - Stake
-      abis:
-        - name: StakingController
-          file: ./abis/StakingController.json
-      eventHandlers:
-        - event: StakeTokenSet(indexed uint256,indexed address)
-          handler: handleStakeTokenSet
-        - event: MinBidSet(indexed uint256,uint256)
-          handler: handleMinBidSet
-        - event: BidAccepted(indexed address,indexed uint256)
-          handler: handleBidAccepted
-        - event: BidClaimed(address,indexed address,indexed uint256,indexed address)
-          handler: handleBidClaimed
-        - event: Bid(indexed address,indexed address,indexed uint256,string,(bytes,string),string,uint256)
-          handler: handleBid
-      file: ./src/controllers/staking.ts
-  - kind: ethereum/contract
-    name: DynamicTokenController
-    network: rinkeby
-    source:
-      address: "0x798190f0640E6089b481298FbAc7EAEC0CFFBEFb"
-      abi: DynamicTokenController
-      startBlock: 7982879
-    mapping:
-      kind: ethereum/events
-      apiVersion: 0.0.4
-      language: wasm/assemblyscript
-      entities:
-        - DynamicTokenAddresses
-      abis:
-        - name: DynamicTokenController
-          file: ./abis/DynamicTokenController.json
-      eventHandlers:
-        - event: DynamicTokenCreated(indexed uint256,indexed address,address,address)
-          handler: handleDynamicTokenCreated
-        - event: DynamicConverterUpgraded(indexed uint256,indexed address,indexed address,address,address)
-          handler: handleDynamicConverterUpgraded
-      file: ./src/controllers/staking.ts
-templates:
-  - kind: ethereum/contract
-    name: DynamicLiquidTokenConverter
-    network: rinkeby
-    source:
-      abi: DynamicLiquidTokenConverter
-    mapping:
-      kind: ethereum/events
-      apiVersion: 0.0.3
-      language: wasm/assemblyscript
-      file: ./src/mappings/core.ts
-      entities:
-        - DynamicToken
-      abis:
-        - name: DynamicLiquidTokenConverter
-          file: ./abis/DynamicLiquidTokenConverter.json
-        - name: Factory
-          file: ./abis/factory.json
-      eventHandlers:
\ No newline at end of file
+        - event: Unpaused(address)
```

Comment:
I know we don't have any subgraph types for this quite yet, is it okay to have events and handlers for events even if we don't handle them in the ts file?

Reply:
I believe so but Ill have to double check. Pretty sure it doesn't really care and just does what the mapping.ts tells it to.